contracts/AtlasMineStakerUpgradeable.sol

```diff
@@ -220,14 +221,11 @@ contract AtlasMineStakerUpgradeable is
      * @param _amount               The amount to withdraw.
      *
      */
-    function withdraw(uint256 depositId, uint256 _amount) public virtual override nonReentrant {
+    function withdraw(uint256 depositId, uint256 _amount) public virtual override whenNotAccruing {
```

Comment:
`nonReentrant` not needed anymore?

Reply:
Hmm, will add this back in

Reply:
> looks good!
> 
> q: how are the `depositIds` calculated/determined? off-chain call from FE (or arbiscan)? that may need some tweaking to handle out of gas errors if we try too many at once

Yep we will need to track previous events in a subgraph so we can see previous accrues that have happened in the same window, and avoid re-accuring. I do think we can add a FE button like we do "stakeScheduled" that cycles through deposits, reading subgraph for the ones that have already accrued. We'll need to call it 5-10 times per window.